process/interface.go

```diff
@@ -481,3 +482,17 @@ type NetworkConnectionWatcher interface {
 	IsConnectedToTheNetwork() bool
 	IsInterfaceNil() bool
 }
+
+// GasHandler is able to perform some gas calculation
+type GasHandler interface {
+	Init()
+	AddGasConsumed(gasConsumed uint64)
+	AddGasRefunded(gasRefunded uint64)
+	SetGasConsumed(gasConsumed uint64)
```

Comment:
SetTotalGasConsumed and SetTotalGasRefunded ? it would be easier to understand

Reply:
done